test/app_test_case.py

```diff
@@ -16,6 +25,37 @@
 from azul_test_case import AzulTestCase
 
 log = logging.getLogger(__name__)
+key_dir = os.path.dirname(__file__)
+public_key_path = os.path.abspath(os.path.join(key_dir, 'public.pem'))
+private_key_path = os.path.abspath(os.path.join(key_dir, 'private.pem'))
+
+
+def generate_test_keys():
+    if not os.path.exists(private_key_path):
+        call(['openssl', 'genrsa', '-out', private_key_path, '2048'])
+        call(['openssl', 'rsa', '-in', private_key_path, '-outform', 'PEM', '-pubout', '-out', public_key_path])
+
+
+def get_test_keys():
+    with open(private_key_path, "rb") as key_file:
+        private_key = serialization.load_pem_private_key(
+            key_file.read(),
+            password=None,
+            backend=default_backend()
+        )
+
+    with open(public_key_path, "rb") as key_file:
+        public_key = serialization.load_pem_public_key(
+            key_file.read(),
+            backend=default_backend()
+        )
+
+    return private_key, public_key
+
+
+def remove_test_keys():
+    os.unlink(public_key_path)
```

Comment:
Method in a test. I don't understand why this deviates from a well established pattern of test classes and methods to global functions.

Reply:
Currently, this is part of a helper class, called `TestKeyManager` in `app_test_case.py`.